modules/migration-upgrading-migration-tool.adoc

```diff
@@ -26,6 +27,40 @@ The following procedure enables you to change the *Manual* approval option to *A
 . Optional: Edit the *Channel*.
 +
 Updating the subscription deploys the updated {mtc-operator} and updates the {mtc-short} components.
+endif::[]
+ifeval::["{mtc-version}" == "1.3"]
```

Comment:
Does this split in MTC version correspond cleanly to different OCP versions? If so, open a separate PR to update the required versions directly.

Reply:
No, it does not. There is no correlation between the MTC and OCP releases.